addons/mail/static/src/core/common/thread_model.js

```diff
@@ -241,25 +242,11 @@ export class Thread extends Record {
              * only the thread type. In particular for chatter, it should depend
              * whether it is displayed in chatter or chat window.
              */
-            return this.type === "chatter" ? 0 : "bottom";
+            return ["mail.box", "discuss.channel"].includes(this.model) ? "bottom" : 0;
```

Comment:
This include is made 3 times in this commit, we might want to have a `isChatter` getter instead, similar to `isChatChannel`.

Reply:
I don't want to introduce an extra getter, the goal of this PR is to remove one.

Also because the split will make this check completely obsolete, and the goal is to being able to easily find usages of `discuss.channel` in code (not through getters).